### PR TITLE
Change image position/height on successful registration page

### PR DESCRIPTION
**Pull request by KhrystynaPavlikovska**
## Description

Fixed image style for correct view in different graphics display resolution.
Bug: https://github.com/ita-social-projects/horondi_client_fe/issues/2170

Also changed position of “welcome” message form.

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
|**image**|**image**|

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/confirmation/confirmation.styles.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ export const useStyles = makeStyles((theme) => ({
   confirmation: {
     background: `url(${BG}) no-repeat `,
     backgroundSize: 'cover',
```
Comment (rostyslaw27) [inline src/pages/confirmation/confirmation.styles.js:7]: write background size and position in background property

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: src/pages/confirmation/confirmation.styles.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ export const useStyles = makeStyles((theme) => ({
...
-    height: '700px',
-    position: 'relative'
+    backgroundPosition: 'center',
+    minHeight: '100vh'
```
Comment (tarasSanotskyi) [inline src/pages/confirmation/confirmation.styles.js:9]: a height 100vh is not a good idea, maybe there will be a scroll that we don't want to see on the page

Comment (KhrystynaPavlikovska) [reply src/pages/confirmation/confirmation.styles.js]: fixed

File: src/pages/confirmation/confirmation.styles.js (hunk 1 above)
Comment (KhrystynaPavlikovska) [reply src/pages/confirmation/confirmation.styles.js]: fixed

**Review by tarasSanotskyi — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by rostyslaw27 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**